Library/ViewModels/PledgeViewCTAContainerViewModelTests.swift

```diff
@@ -98,10 +122,10 @@ internal final class PledgeViewCTAContainerViewModelTests: TestCase {
     self.vm.inputs.configureWith(value: pledgeData)
 
     self.submitButtonIsHidden.assertValues([false])
-    self.applePayButtonIsHidden.assertValues([false])
+    self.applePayButtonIsHidden.assertValues([true])
```

Comment:
I was wondering about this - didn't we use to support Apple Pay in Change Payment Method? 🤔

Reply:
Ah, good catch - yes, will update.